site/docs/v1/tech/identity-providers/samlv2/okta-saml.adoc

```diff
@@ -0,0 +1,107 @@
+---
+layout: doc
+title: SAML v2 with Okta
+description: Configure SAML v2 for Okta
+navcategory: login-methods
+---
+
+:sectnumlevels: 0
+
+== Configure SAML v2 for Okta
+
+This documentation will guide you in configuring SAML v2 IdP for Okta. In this case, FusionAuth will act as Service Provider (SP) to Okta (IdP). Functionally, the result will allow you to display a [uielement]#Login with Okta# button on your FusionAuth application's login page and connect via SAML to Okta users/applications.
+
+image:identity-providers/okta-saml/okta-saml-result.png[result-intro,width=1200,role=bottom-cropped]
+
+== Prerequisites
+* A FusionAuth instance running on a publicly accessible URL. You can spin up a link:/pricing[basic FusionAuth Cloud instance] or link:/docs/v1/tech/installation-guide/[install it on any server].
+* An link:https://www.okta.com/free-trial/[Okta Workforce Identity Cloud account] connected to a business email address.
+** The link:https://help.okta.com/eu/en-us/Content/Topics/end-user/ov-overview.htm?cshid=csh-user-ov-overview[Okta Verify app] if your Okta account is not configured to send SMS messages for 2FA.
+
+== Okta Configuration
+
+Log in to Okta and navigate to the [uielement]#Admin# panel.
```

Comment:
```suggestion
Log in to Okta and navigate to the [breadcrumb]#Admin# panel.
```

If you are navigating someplace, use breadcrumb. If it is a field, use field. If it is in the UI but anything else, use uielement.